src/plugins/vis_type_timeseries/server/lib/search_strategies/capabilities/default_search_capabilities.ts

```diff
@@ -13,26 +13,54 @@ import {
   getSuitableUnit,
 } from '../../vis_data/helpers/unit_to_seconds';
 import { RESTRICTIONS_KEYS } from '../../../../common/ui_restrictions';
+import {
+  TIME_RANGE_DATA_MODES,
+  PANEL_TYPES,
+  METRIC_AGGREGATIONS,
+  SIBLING_PIPELINE_AGGREGATIONS,
+} from '../../../../common/enums';
+import type { Panel } from '../../../../common/types';
 
 export interface SearchCapabilitiesOptions {
   timezone?: string;
   maxBucketsLimit: number;
+  panel?: Panel;
```

Comment:
do we have any cases when this argument is undefined?

Reply:
When we getting fields we should get search strategy, this operation also request capabilities but we don't pass panel in request.